.changeset/fair-grapes-joke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Register the `TechDocs` addons on the catalog page and also test their rendering in `EntityDocs` sub pages.
```

Comment:
Let's do the usual highlighting of the changes that need to be made to keep existing apps up to date. Perhaps describe it at a slightly higher level, just what elements to replace with what, and that it can all be stored in a constant too

Reply:
Done ✔️